std_msgs/Header header 
uint8 severity 
geometry_msgs/Point position